app/response_svc.py

```diff
@@ -148,3 +153,30 @@ async def _apply_adversary_config(self):
     async def _save_configurations(self):
         with open('plugins/response/conf/response.yml', 'w') as config:
             config.write(yaml.dump(self.get_config(name='response')))
+
+    def _filter_ability_facts(self, unique_facts, relationships, red_pid, original_pid):
```

Comment:
yes, I'd move to planner, tying it closer to the decision making seems like it will be better down the road.

Reply:
after further discussion let's add in as is now and map out conversion to a response planner starting next sprint